### PR TITLE
1.11.0

### PR DESCRIPTION
**Pull request by lydell**
I'd love feedback on the first two commits (CLI flags update and changelog). The plan is to release on Saturday.

@jashkenas @GeoffreyBooth @vendethiel

### CODE REVIEW

File: documentation/index.html.js [hunk 1]
```diff
@@ -1269,6 +1269,100 @@ Expressions
...
+              <code>yield return</code> can no longer mistakenly be used as an expression.
+            </li>
+            <li>
+              <p>
+                <code>yield</code> now mirrors <code>return</code> in that it
+                can be used stand-alone as well as with expressions. Where you
+                previously wrote <code>yield undefined</code>, you may now write
+                simply <code>yield</code>. However, this means also inheriting
+                the same syntax limitations that <code>return</code> has, so
+                these examles no longer compile:
```
Comment (GeoffreyBooth) [inline documentation/index.html.js]: `examples`

File: documentation/index.html.js [hunk 2]
```diff
@@ -1269,6 +1269,100 @@ Expressions
...
+              <code>yield</code> no longer present.
+            </li>
+          </ul>
+        </li>
+        <li>
+          Improved several error messages.
+        </li>
+        <li>
+          Just like <code>undefined</code> compiles to <code>void 0</code>,
+          <code>Nan</code> now compiles into <code>0/0</code> and
```
Comment (GeoffreyBooth) [inline documentation/index.html.js]: `Nan` or `NaN`?

**Comment by GeoffreyBooth** [comment]
Other than those notes, looks good to me. Wow, we really added all those flags? 😄

**Comment by jashkenas** [comment]
I didn't do a review of actual-changes-vs-changelog — but looks good to me!

**Comment by vendethiel** [comment]
:+1: nicely done

**Comment by lydell** [comment]
1.11.0 is now published on npm, and coffeescript.org has been updated.

**Comment by GeoffreyBooth** [comment]
Nice work! Thanks @lydell!

**Comment by lydell** [comment]
@GeoffreyBooth And thank _you_ for all your awesome work!

**Pull request merged**